app/views/layouts/_footer.html.erb

```diff
@@ -1,58 +1,55 @@
-            <% end %>
-            <%= link_to 'Funding and Programmes', funding_home_path -%> | 
-            <% if !Seek::Config.funding_link.blank? %>
-              <a href="<%=Seek::Config.funding_link%>" target="_blank">
-                <%=Seek::Config.funding_link%>
-              </a> |
-            <% end %>
-            <%= link_to 'Credits', Seek::Help::HelpDictionary.instance.help_link(:credits), :target => '_blank' -%>
-            <% if Seek::Config.terms_enabled %>
-                | <%= link_to("Terms & Conditions", terms_home_path) %>
-            <% end %>
-            <% if Seek::Config.privacy_enabled %>
-                | <%= link_to("Privacy Policy", privacy_home_path) %>
-            <% end %>
-            <% if Seek::Config.imprint_enabled %>
-                | <%= link_to("Imprint", imprint_home_path) %>
-            <% end %>
-            <% if !Seek::Config.cite_link.blank? %>
-              | <%= link_to "Cite us", Seek::Config.cite_link, :target => '_blank' -%>
-            <% end %>
-            <% if !Seek::Config.contact_link.blank? %>
-              | <%= link_to "Contact us", Seek::Config.contact_link, :target => '_blank' -%>
-            <% end %>
-            <% if cookie_consent.required? %>
-              | <%= link_to 'Cookie preferences', cookies_consent_path %>
-            <% end %>
-          </div>
-          <div class="ft-info">
-            Copyright &copy; 2008 - 2023
-            <%= link_to "The University of Manchester", "http://www.manchester.ac.uk/", :target => '_blank' %> and
-            <%= link_to "HITS gGmbH", "http://www.h-its.org/", :target => '_blank' %>
-            <% if Seek::Config.copyright_addendum_enabled  %>
-                <br/><%= Seek::Config.copyright_addendum_content.html_safe%>
-            <% end %>
-          </div>
-        <% end -%>
+<div id="ft">
+  <div class="container">
+    <div id="seek-credit">
+      <a href="https://seek4science.org/" target="_blank" class="powered-by">
+        Powered by<br>
+        <%= image(:seek_logo, width: 64, height: 64) %>
+      </a>
+      <span class="subtle"><%= version_text -%></span>
+    </div>
+    <% cache [logged_in?, Settings.global] do -%>
+      <div class="ft-info">
+        <% if (Seek::Config.instance_admins_link != Seek::Config.instance_link) && (Seek::Config.instance_admins_name != Seek::Config.instance_name) %>
+          <%= link_to "About #{Seek::Config.instance_admins_name}", Seek::Config.instance_admins_link, target: '_blank' %> |
+        <% end %>
+        <%= link_to "About #{Seek::Config.instance_name}", Seek::Config.instance_link, target: '_blank' %> |
+        <% if logged_in_and_registered? %>
+          <%= link_to 'Provide feedback', feedback_home_path -%> |
+        <% end %>
+        <%= link_to 'Funding and Programmes', funding_home_path -%> |
+        <% if Seek::Config.funding_link.present? %>
+          <a href="<%=Seek::Config.funding_link%>" target="_blank">
```

Comment:
this should be changed to `link_to` (no idea why it was written this way). I don't think link_to automatically adds the noreferrer though. We may need to update all links that open in a new tab.

Reply:
i'll just change it to `link_to` for now, we can think about how to update all external links later